homeassistant/components/knx/light.py

```diff
@@ -142,6 +146,113 @@ def _device_unique_id(self) -> str:
             f"{self._device.blue.brightness.group_address}"
         )
 
+    @staticmethod
+    def _create_light(xknx: XKNX, config: ConfigType) -> XknxLight:
+        """Return a KNX Light device to be used within XKNX."""
+
+        def create_light_color(
+            color: str, config: ConfigType
+        ) -> tuple[str | None, str | None, str | None, str | None]:
+            """Load color configuration from configuration structure."""
+            if "individual_colors" in config and color in config["individual_colors"]:
```

Comment:
Invert this check and return the tuple with None if true. Then we can outdent below.

Reply:
I replaced this with afunction returning one address-configuration at a time. This saves some extra lines of code packing and unpacking these tuples.